osquery/core/flags.cpp

```diff
@@ -115,7 +115,11 @@ std::string Flag::getDescription(const std::string& name) {
 }
 
 Status Flag::updateValue(const std::string& name, const std::string& value) {
-  if (instance().flags_.count(name) > 0) {
+  auto it = instance().flags_.find(name);
+  if (it != instance().flags_.end()) {
+    if (it->second.cli) {
```

Comment:
What is the intent of this code?

Reply:
if it is CLI_FLAG then dont change the value via config refresh.

Reply:
This is a good idea, if someone adds a CLI-only flag to the config we should emit a warning line.